### DISCUSSION TITLE
is it possible to add a line item without a supplier part

### DISCUSSION THREAD

**Discussion by ATLAS2246** [inventree/InvenTree#7885] [Q&A] [answered]
So im using inventree as a stock system where i am issuing items and then they get returned when someone leaves my organisation. Im using sales orders to issue the items and then purchase orders to return the items.

I'd like to be able to add line items to a purcahse order without having to add a supplier part for each line item and i wasnt sure if there was a way to do that.

**Comment by SchrodingersGat** [comment, marked as answer]
You need a way to map the internal part to the "supplier code" from the particular supplier. That is what the SupplierPart achieves.

Simple answer: you'll need to add a supplier part before you can add it to the purchase order!

**Reply by ATLAS2246** [reply]
all good, i was hoping id be able to skip that step in some way but it doesnt add that much time
